tests/test_lab_dev/test_transformations.py

```diff
@@ -477,6 +478,50 @@ def test_representation(self):
         assert math.allclose(rep2.c, [1.0 + 0.0j])
 
 
+class TestAmplifier:
```

Comment:
Let's add a test or two to verify correct operation. You can use identities for e.g. how an amplifier maps a coherent state, or how the Amplifier composes with the Attenuator.

Reply:
I added a test for composing the amplifier and attenuator